wp-content/themes/core/components/card/css/card.pcss

```diff
@@ -157,6 +159,12 @@
 	@mixin t-body-small;
 
 	color: var(--color-grey);
+
+	/* CASE: Dark Theme */
+	.t-theme--dark & {
+		color: var(--color-white);
+		opacity: 0.8;
```

Comment:
Why is the whole card's opacity set here? Wouldn't we only want to adjust the background color opacity?